### PR TITLE
Add Twitter link

### PR DESCRIPTION
**Pull request by robrich**
... except I didn't sign-off?

### CODE REVIEW

**Comment by LucilleH** [comment]
I don't think we can contribute directly to devbox repo yet. Right now we go through /axiom

**Comment by loreto** [comment]
Yeah, for now go through axiom.

**Comment by robrich** [comment]
@loreto Now that it isn't tied to axiom, re-open and merge?

File: README.md [hunk 1]
```diff
@@ -153,6 +153,7 @@ Want more languages? [Ask for a new Language](https://github.com/jetpack-io/devb
 
 + Chat with us by joining the [Jetpack.io Discord Server](https://discord.gg/agbskCJXk2) – we have a #devbox channel dedicated to this project. 
 + File bug reports and feature requests using [Github Issues](https://github.com/jetpack-io/devbox/issues)
++ Follow us on [Jetpack's Twitter](https://twitter.com/jetpack_io) for product updates
```
Comment (loreto) [inline README.md:156]: I would do:
```suggestion
+ Follow [@jetpack_io](https://twitter.com/jetpack_io) on Twitter for product updates.
```

**Review by loreto — APPROVED** [review]
LGTM. See suggestion

**Pull request merged**